### PR TITLE
Update ctera acl documentation and tests.

### PR DESCRIPTION
**Pull request by chenkins**

### CODE REVIEW

**Comment by chenkins** [comment]
Depends on #15956

**Review by ylangisc — APPROVED** [review]

File: ctera/src/main/java/ch/cyberduck/core/ctera/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Custom Properties in namespace `http://www.ctera.com/ns` in DAV Resources to Support NT-ACL and WORM Data
 
-## Preflight Checks
+## Preflight Checks (nfs 4.x)
```
Comment (dkocher) [inline ctera/src/main/java/ch/cyberduck/core/ctera/README.md:3]: Preflight checks apply to all connect modes not dependent on filesystem driver.

File: ctera/src/main/java/ch/cyberduck/core/ctera/README.md [hunk 2]
```diff
@@ -31,27 +42,26 @@ N.B. no need to check `readpermission` upon mv/cp.
 
 N.B. `x` on files is only set for POSIX backends, i.e. never for CTERA.
 
-### macOS File Provider Capabilities
+### macOS File Provider Capabilities (mode integrated) 5.x
```
Comment (dkocher) [inline ctera/src/main/java/ch/cyberduck/core/ctera/README.md:45]:
```suggestion
### macOS File Provider Capabilities (_Integrated_ connect mode) Mountain Duck 5+
```

File: ctera/src/main/java/ch/cyberduck/core/ctera/README.md [hunk 3]
```diff
@@ -31,27 +42,26 @@ N.B. no need to check `readpermission` upon mv/cp.
...
 N.B. File Provider sets the `x` flag on all folders independent of `NSFileProviderFileSystemUserExecutable`.
+N.B. `x` on files is only set for POSIX backends, i.e. never for CTERA.
 
 #### Documentation
 
 * https://developer.apple.com/documentation/fileprovider/nsfileproviderfilesystemflags
 * https://developer.apple.com/documentation/fileprovider/nsfileprovideritemcapabilities
 
-### Windows ACLs
+### Windows CBFS API (mode sync) and Cloud Files API (mode integrated) (5.x)
```
Comment (dkocher) [inline ctera/src/main/java/ch/cyberduck/core/ctera/README.md:64]:
```suggestion
### Windows CBFS API (_Online_ and _Smart Synchronization_ connect mode) and Cloud Files API (_Integrated_ connect mode) Mountain Duck 5+
```

**Review by dkocher — CHANGES_REQUESTED** [review]

**Review by ylangisc — APPROVED** [review]

**Review by dkocher — CHANGES_REQUESTED** [review]
It looks to me there is a lot of duplication. It would suffice to sumarize the mapping to POSIX & FileSystemRights model and detail how dynamic preflight checks are used for filesystem operations.

**Comment by chenkins** [comment]
Superseded by #16111

**Pull request closed without merging**